src/cdk/a11y/live-announcer.ts

```diff
@@ -14,7 +14,7 @@ import {
   SkipSelf,
   OnDestroy,
 } from '@angular/core';
-import {Platform} from '@angular/cdk/platform';
+import {DOCUMENT} from '@angular/platform-browser';
```

Comment:
The `DOCUMENT` from platform-browser is deprecated; should use the one from `@angular/common` instead